python/hsfs/feature_view.py

```diff
@@ -2097,6 +2344,9 @@ def get_train_test_split(
         self,
         training_dataset_version,
         read_options: Optional[Dict[Any, Any]] = None,
+        primary_keys=False,
+        event_time=False,
```

Comment:
why they are needed when there is already `training_helper_columns`?
shouldn't users add primary key and event time to the helper columns?